eng/pipelines/tagger.yml

```diff
@@ -0,0 +1,90 @@
+
+# Name: DotNet-Project-System-Tagger
+# URL: https://dev.azure.com/devdiv/DevDiv/_build?definitionId=17277
+
+# Creates a Git tag in our repo when a VS Insertion PR is merged.
+
+###################################################################################################################################################################
+# PIPELINE METADATA
+###################################################################################################################################################################
+
+# Disable CI builds for this pipeline.
+# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/trigger?view=azure-pipelines#examples
+trigger: none
+
+# Disable PR builds for this pipeline.
+# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#examples
+pr: none
+
+resources:
+  # https://stackoverflow.com/a/63270937/294804
+  repositories:
+  - repository: self
+    name: project-system
+    type: git
+    ref: feature/InsertionPRDescriptionAndTagging
+  - repository: VS
+    name: VS
+    type: git
+    ref: main
+    trigger:
+      branches:
+        include:
+        - main
+      paths:
+        include:
+        # File containing the package version for Managed/Managed.VS packages.
+        - src/ConfigData/Packages/ProjectSystem/Managed.props
+        # File containing the package version for AppDesigner/Editors packages.
+        - src/ConfigData/Packages/Wizard/packages.props
+
+pool:
+  # Agent Queue: https://devdiv.visualstudio.com/DevDiv/_settings/agentqueues?queueId=3123&view=jobs
+  name: VSEngSS-MicroBuild2022-1ES
+  # Demands Docs: https://docs.microsoft.com/azure/devops/pipelines/process/demands?view=azure-devops&tabs=yaml#manually-entered-demands
+  demands: Cmd
+
+variables:
+  # https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/26284/Enabling-SBOM-For-Your-Component-Insertion-into-VS?anchor=1.-add-the-%27manifest-generator-task%27-to-your-pipeline
+  Packaging.EnableSBOMSigning: true
+  # Opt out of automatically injecting Codesign Validation into the pipeline. We run Codesign Validation as part of the Compliance pipeline.
+  # See: https://aka.ms/gdn-injection
+  runCodesignValidationInjection: false
+  # Suspend enforcement of NuGet Single Feed Policy. See:
+  # - https://aka.ms/nugetmultifeed
+  # - https://docs.opensource.microsoft.com/tools/nuget_security_analysis/nuget_security_analysis/
+  # - https://docs.opensource.microsoft.com/tools/cg/how-to/nuget-multifeed-configuration/
+  # - https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/5205/TSG-Build-Broken-Due-to-Using-Multiple-Feeds?anchor=setting-nugetsecurityanalysiswarninglevel-in-cdp
+  NugetSecurityAnalysisWarningLevel: none
+
+parameters:
+- name: VsCommitId
+  displayName: VS Commit ID
+  type: string
+  default: $(Build.SourceVersion)
```

Comment:
This might not work and needs tested.

Reply:
Replaced this mechanism with a 'Default' value that conditionally sets a variable to `Build.SourceVersion`. There is no reason for me to believe this won't work when triggered by a VS insertion commit.